comps/dataprep/src/integrations/redis.py

```diff
@@ -282,54 +310,26 @@ class OpeaRedisDataprep(OpeaComponent):
 
     def __init__(self, name: str, description: str, config: dict = None):
         super().__init__(name, ServiceType.DATAPREP.name.lower(), description, config)
-        self.client = redis.Redis(connection_pool=redis_pool)
-        self.data_index_client, self.key_index_client = asyncio.run(self._initialize_client())
-        self.embedder = asyncio.run(self._initialize_embedder())
```

Comment:
What is the reason for the embedder initialization moving from here to `ingest_chunks_to_redis`? Since `embedder` is no longer a class property, won't it get re-instantiated each time `ingest_chunks_to_redis` is called?